modin/experimental/cloud/ray-autoscaler.yml

```
@@ -120,7 +120,8 @@ setup_commands:
         source ~/.bashrc
         conda create --name modin --yes
         conda activate modin
-        conda install --yes --override-channels -c intel/label/validation -c conda-forge modin {{CONDA_PACKAGES}}
+        conda install --yes --override-channels -c intel/label/validation -c conda-forge {{CONDA_PACKAGES}}
```

```suggestion
        conda install --yes -c intel/label/validation -c conda-forge {{CONDA_PACKAGES}}
```
we need this `--override-channels` dropped as new version of conda package for Modin requires `libffi >= 3.3` which is available in `main` channel only